tests/drivers/fuel_gauge/sy24561/src/test_sy24561.c

```diff
@@ -63,6 +63,10 @@ ZTEST_USER_F(sy24561, test_get_props__returns_ok)
 	union fuel_gauge_prop_val props[ARRAY_SIZE(prop_types)];
 
 	zassert_ok(fuel_gauge_get_props(fixture->dev, prop_types, props, ARRAY_SIZE(props)));
+	zassert_equal(props[0].voltage, 3199000);
+	zassert_equal(props[1].relative_state_of_charge, 74);
```

Comment:
These do not strictly match the values in the emulator code due to errors from integer division. Still, they help us see the driver still decodes props using appropriate formulae